deps/v8/build/toolchain.gypi

```diff
@@ -1104,7 +1104,7 @@
           'VCCLCompilerTool': {
             'Optimization': '0',
             'conditions': [
-              ['component=="shared_library"', {
+              ['component=="shared_library" or node_shared=="true"', {
```

Comment:
Perhaps make this a bit more generic: something like `force_dynamic_crt` rather than `node_shared` and set that in node.gyp based on `node_shared`.